libftl/ftl-sdk.c

```diff
@@ -15,6 +15,11 @@ FTL_API const int FTL_VERSION_MAINTENANCE = 3;
 FTL_API ftl_status_t ftl_init() {
   ftl_init_sockets();
   ftl_logging_init();
+#ifndef _WIN32
+  pthread_mutexattr_init(&ftl_default_mutexattr);
+  // Set pthread mutexes to recursive to mirror Windows mutex behavior
```

Comment:
Ideally, we shouldn't be relying on recursive mutex locking unless we have a good reason, but we are right now (media_send_audio and _media_send_packet) and I didn't want to change code behavior for this change. Something to consider.